x-pack/plugins/enterprise_search/public/applications/app_search/components/curations/views/curation_suggestion/curation_result_panel.scss

```diff
@@ -0,0 +1,24 @@
+.curationResultPanel {
+  border-radius: $euiSizeM;
+  margin-top: $euiSizeS;
+  padding: $euiSizeXS;
+}
+
+.curationResultPanel--current, .curationResultPanel--promoted {
+  border: 2px solid $euiColorPrimary;
```

Comment:
nit: there are variables for border widths... `$euiBorderWidthThick` is 2px

Reply:
Thanks for pointing that out! d25a72e18f53b79d4395a7e468fb464b9296d6e8